src/modules/getPokemones.js

```diff
@@ -0,0 +1,39 @@
+const getPokemones = async () => {
+  const config = {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  };
+
+  const resources = [];
+  const promises = [];
+
+  for (let i = 1; i <= 10; i += 1) {
+    const url = `https://pokeapi.co/api/v2/pokemon/${i}`;
+    promises.push(
+      fetch(url, config)
+        .then((response) => response.json())
+        .then((data) => {
```

Comment:
- [OPTIONAL] The same here, Instead of relying on Promises (using .then, .then), consider incorporating the await keyword. Kindly make the necessary adjustment. 👍🏻